src/style.css

```diff
@@ -1 +1,64 @@
-/* add styles here */
+body {
+  margin: 0;
+  padding: 0;
+
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  background-image: url(images/star.svg);
+  width: 16px;
+  height: 16px;
+  margin-right: 4px;
+  background-repeat: no-repeat;
+  background-position: center;
+
+}
+
+.stars--1 :nth-child(-n + 1) {
+  background-image: url(images/star-active.svg);
+  width: 16px;
+  height: 16px;
+  margin-right: 4px;
+  background-repeat: no-repeat;
+  background-position: center;
```

Comment:
You don't need to duplicate all styles. Just change the property that should be changed.
Also, you can combine the selectors with coma